projects/eventonica/eventonica-part6-postgres.md

```diff
@@ -1,9 +1,51 @@
 # Eventonica - Part 6 - Postgres Database
 
 ### Overview
-
+Previously, your data was stored in memory in Express, so your data would disappear if the application restarted. For production applications, data must be stored in a more permanent way. In this lesson, you will move your EventRecommender data to a Postgres database and connect to that database in your Express APIs.
 
 ### Instructions
 
+1. Ensure that you have Postgres installed on your machine - see instructions [here](../../databases/installing-postgresql.md).
+
+1. Create a new database named `eventonica`.
+
+1. In your `eventonica` database, create a table named `users` that contains the same fields as your `User` class in `eventRecommender.js`. Make the `id` column a [primary key](https://www.postgresql.org/docs/12/ddl-constraints.html#DDL-CONSTRAINTS-PRIMARY-KEYS) and use the datatype [serial](https://www.postgresql.org/docs/12/datatype-numeric.html#DATATYPE-SERIAL) to create an auto-incrementing integer id. Try running the following SQL insert multiple times to see how the `serial` type works: `INSERT INTO users (name) values ('jane');`. Your table should have automatically filled the `id` field for you!
+
```

Comment:
Would you mind breaking up this longer step into sub-steps

If you have time, including a DDL snippet of the table might be nice too

Reply:
What do you mean by the DDL snippet?

Reply:
Sorry for using jargon unnecessarily. I believe it's the official name for the sql you can use to create the table, e.g.

```sql
CREATE TABLE SCOTT.DEPARTMENT
(DEPARTMENT_ID NUMBER,
   DNAME VARCHAR2 (100),
   LOCATION_ID NUMBER DEFAULT 100);
```

But now that I'm thinking about it, I guess it depends if we want that to be part of what they need to figure out on their own, so maybe ignore that suggestion!